src/factorizations.jl

```diff
@@ -161,11 +213,9 @@ function eigvals(
   A::AbstractArray, biperm::BlockedPermutation{2}; ishermitian=nothing, kwargs...
```

Comment:
```suggestion
  A::AbstractArray, biperm::BlockedPermutation{2}; kwargs...
```
I think this was discarding the `ishermitian` flag and not passing it on before